distribution/docs/src/main/resources/_contents/_core-concepts/registry-intro-contents.adoc

```diff
@@ -0,0 +1,4 @@
+
+== Registry
+
+${ddf-registry} is a
```

Comment:
@vinamartin will probably have some additional material, but you can add:

The DDF Registry Application provides a method for creating and configuring connections to/from registry nodes. Each registry has the capability to serve as an index of nodes and can be used to query across a network of DDFs.